orca-core/src/test/groovy/com/netflix/spinnaker/orca/pipeline/persistence/ExecutionRepositoryTck.groovy

```diff
@@ -625,9 +625,9 @@ class JedisExecutionRepositorySpec extends ExecutionRepositoryTck<JedisExecution
         context = [:]
       }
     }
+    def previousRepository = new JedisExecutionRepository(new RedisClientSelector([new JedisClientDelegate("primaryDefault", jedisPoolPrevious)]), 1, 50)
```

Comment:
This looks more like how `previousRepository` should be defined in the above "can store an orchestration already in previousRedis back to previousRedis" test